contracts/InternalContract.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity >=0.4.15;
+
+contract SponsorWhitelistControl {
```

Comment:
Redundant definition in SponsorFaucet.sol

Reply:
rm and use import way

Reply:
rm and use import way